x-pack/docs/en/rest-api/security/create-users.asciidoc

```diff
@@ -73,7 +73,7 @@ The following parameters can be specified in the body of a POST or PUT request:
 (object) Arbitrary metadata that you want to associate with the user.
 
 `password`::
-(string) The user's password. Passwords must be at least 6 characters long.
+(Required, string) The user's password. Passwords must be at least 6 characters long.
```

Comment:
In some other APIs where a parameter is required in some but not all situations, we've added an asterisk like this:
```suggestion
(Required*, string) The user's password. Passwords must be at least 6 characters long.
```
E.g. https://www.elastic.co/guide/en/elasticsearch/reference/7.9/analysis-dict-decomp-tokenfilter.html